src/components/Event.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+
+class Event extends Component {
+  constructor (props) {
+    super();
+    this.state = {
+      className: props.className,
```

Comment:
It's not necessary to assign props to state. Most likely this component can be converted to stateless component.

Reply:
yes, but i'll change this component in the next PR. This is just a preparation for the next step.